Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/UnityUIInteractables/UnityUIToggleButton.prefab

```diff
@@ -165,5 +162,6 @@ PrefabInstance:
       propertyPath: m_text
       value: Say "Select"
       objectReference: {fileID: 0}
-    m_RemovedComponents: []
+    m_RemovedComponents:
+    - {fileID: 5168285722132125214, guid: f15ddb9bd0bf19d478c51ea991c6a7ce, type: 3}
```

Comment:
Is this something that could be removed from the base prefab as well, or is its removal specific to this prefab variant?

Reply:
hmmm this is weird - i removed that component from the prefab as well - but might be related to the order i did it? - so it's definitely gone from the prefab (it's the touchhandler)
I'll revert that change and see if it just poofs away

Reply:
had to manually remove it from file - i think unity got confused because I deleted it in the derived prefab first and then in the base prefab - i didn't see at first that there was another hierarchy level